server/base/src/main/java/org/apache/accumulo/server/cli/ClientOpts.java

```diff
@@ -34,6 +32,10 @@ public Instance getInstance() {
     if (instance == null) {
       return HdfsZooInstance.getInstance();
     }
-    return new ZooKeeperInstance(this.getClientConfiguration());
+    try {
+      return getConnector().getInstance();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
Same here, with respect to bad exception handling.

Reply:
Fixed now